src/components/Navbar.tsx

```diff
@@ -105,7 +105,7 @@ function Navbar({ scrollToExplore }: { scrollToExplore: any }) {
             preview={false}
             style={{
               paddingRight: screens.md ? "1.5em" : "10px",
-              height: "26px",
+              height: "26px", maxWidth: screens.md ? '184.17px' : '36.67px',
```

Comment:
This is fine for now. @Vinyl-Davyl @nitro56565 can we please look into having a consistent unit for styles, we are using em, px both. Let's please fix this as well.